modulemd/modulemd-simpleset.c

```diff
@@ -374,3 +374,54 @@ modulemd_simpleset_new (void)
 {
   return g_object_new (MODULEMD_TYPE_SIMPLESET, NULL);
 }
+
+/**
+ * modulemd_simpleset_is_equal:
+ * @other: A #ModulemdSimpleSet to compare against
+ *
+ * Returns: True if the sets contain the same strings.
+ *
+ * Since: 1.3
+ */
+gboolean
+modulemd_simpleset_is_equal (ModulemdSimpleSet *self, ModulemdSimpleSet *other)
+{
+  gboolean ret = FALSE;
+  gchar **set_a = NULL;
+  gchar **set_b = NULL;
+  gsize i = 0;
+
+  if (modulemd_simpleset_size (self) != modulemd_simpleset_size (other))
+    {
+      /* If they have a different number of strings in the set, they can't
+       * be identical.
+       */
+      ret = FALSE;
+      goto done;
+    }
+
+  set_a = modulemd_simpleset_dup (self);
```

Comment:
why not simply `g_auto (GStrv) set_a ...`

Reply:
Didn't think of it. Updated.